### PR TITLE
Add spacing setting for formatting

### PR DESCRIPTION
**Pull request by xorye**
Added formatting spacing setting for https://github.com/redhat-developer/quarkus-ls/pull/110

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

**Comment by xorye** [comment]
I've set the default to false (as in, don't add spaces around equals sign), since we didn't have spaces around the equals sign for completion.

**Comment by angelozerr** [comment]
@xorye  could you resolve the conflicts please.

File: package.json [hunk 1]
```diff
@@ -157,10 +157,16 @@
             }
           }
         },
+        "quarkus.tools.formatting.insertSpaces": {
```
Comment (angelozerr) [inline package.json]: Should not be `quarkus.tools.formatting.insertSpacesBetweenAssign` ?

Comment (fbricon) [reply package.json]: *.surroundEqualsWithSpaces ?

Comment (xorye) [reply package.json]: fixed

Comment (angelozerr) [reply package.json]: @xorye it seems @fbricon would prefer `quarkus.tools.formatting.surroundEqualsWithSpaces`? If it that please do the fix and do the same thing for Quarkus LS settings.

Comment (xorye) [reply package.json]: fixed with @fbricon 's suggestion

**Comment by xorye** [comment]
The setting has now been updated to: `quarkus.tools.formatting.surroundEqualsWithSpaces`

**Pull request merged**